### ISSUE TITLE
[Bug] [a11y] Increase size of elements in bottom section of BrowserToolbar

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#7756]
## Prerequisites
1. Make sure you have Accessibility Scanner installed and turned on.
2. Make Fenix the default browser app.

## Steps to reproduce
1. Open an external browsing app (i.e. Gmail), and open a link to a website.
2. Using the Accessibility Scanner, scan the Custom tab's three-dot menu overlay.

### Expected behavior
No major discrepancies are reported.

### Actual behavior
The back, forward and refresh items are suggested to be increased from 24dp to 48 dp or larger.

### Device information
* Android devices: Huawei Honor 8 (Android 7), Samsung Galaxy Tab S3 (Android 8)
* Fenix version: 01/16 Nightly build

![Screenshot_20200116-121341_Accessibility Scanner](https://user-images.githubusercontent.com/40694158/72516690-c9ad2080-385a-11ea-8cae-1ffa67394640.jpg)

**Comment by sblatz** [comment]
This should be a relatively quick fix, but may live in AC.

**Comment by Diana-Rus** [comment]
Hi, issue still reproduces with Samsung Galaxy S9 (Android 8) on Nightly 4/7 (Build #20980610).

**Comment by mcarare** [comment]
This was fixed This was fixed, issue was reported for normal tab (https://github.com/mozilla-mobile/fenix/issues/10359), and both menus use the same buttons from AC.
